docs/manifest.json

```diff
@@ -104,7 +104,7 @@
       ["<UserProfile />", "/components/user/user-profile"],
       "# Guides",
       ["Sync data to your backend", "/users/sync-data"],
-      ["Web3 authentication", "/users/web3-wallets"]
```

Comment:
❓ `Web3 authentication` feels more descriptive than just `Web3`. Why the change?